protocols/Emory_University/README.md

```diff
@@ -0,0 +1,45 @@
+# PCR/qPCR prep
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+### Partner
+Emory University
+
+## Categories
+* Sample Prep
+	* PCR
+
+## Description
+This protocol allows the robot to transfer KAPA Taq ReadyMix, primers and DNA samples into a target plate. Two pipettes are used in this protocol: P10-Single and P10-Multi.
```

Comment:
Description is good, move the 'steps' in the process section. Again remove partner